### ISSUE TITLE
Issues with OMSTI sensekeys and POS tags

### ISSUE THREAD

**Issue by mbevila** [getalp/UFSAC#1]
First of all, thank you for your work!
I noticed a few issues at occuring systematically at least within the OMSTI corpus. All the WN sense keys with POS 'adjective satellite' (hence of the form word%5...) always appear as word%3, as if they were simple adjectives. You always find typical%**3**:00:00:normal:01 and never the correct typical%**5**:00:00:normal:01 (I double checked the dict/index.sense file in WN 3.0). 
It confuses me a little that WN 3.1 online (see the third sense of _typical_ http://wordnetweb.princeton.edu/perl/webwn?c=6&sub=Change&o2=&o0=1&o8=1&o1=1&o7=1&o5=&o9=&o6=&o3=&o4=&i=-1&h=000&s=typical) also uses %3 in place of %5. Also some of the POS Tags, such as 'n', 'v', 'a', '&apos;' and '&apos;&apos;' are not valid PTB.

**Comment by loic-vial** [comment]
Hello and thank you for your interest in our work :)

About the adjective satellite marker in WordNet, we detected an inconsistency in the original data of version 3.0 between the WordNet raw "dict" directory, where adjective satellite have the "%5" marker, and the "glosstag" directory, where they all have the "%3" marker.

I remember seeing somewhere in the WordNet 3.0 doc, or maybe in a changelog from WN 2.1 to WN 3.0, a paragraph saying that adjective satellite will now have the same "%3" marker as all the other adjectives, starting with version 3.0.

And as you said, in online version 3.1, there is no more %5 marker, so we decided to replace all %5 marker in OMSTI (and all other corpora) to %3, so there is no more confusion: an adjective always has a %3 marker in UFSAC.
There is no clash with existing "%3" adjectives so it's just simpler and less confusing like that ^^

About the POS tags, we are aware and we will release soon a fixed version of the resource with a guarantee that all tags are valid PTB. The problem was that for some corpora (like semeval 2015 task 13), the original data had a "N/V/A/R/X" tags, and when those tags already existed in the original version, we did not want to override them in the UFSAC version. But we have a smarter conversion algorithm now so this problem is no more.
